src/tests/test_buildercore_terraform.py

```diff
@@ -82,3 +85,7 @@ def test_fastly_template_complex(self):
             },
             data
         )
+
+    def _parse_template(self, terraform_template):
+        """use yaml to load JSON due to https://stackoverflow.com/a/16373377/91590"""
```

Comment:
due to url? I know what you're getting at, I'd like an actual explanation with the url as reference please